lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -27,7 +27,7 @@ def run!(args)
     end
 
     def analyze
-      @matches = @output.chomp.split("\n\n").map { |m| m.split("\n") }
+      @matches = @output.chomp.split("\n\n").map { |m| m.gsub(/\n\s+\[/, "SPLIT_ME_HERE_PLEASE[").split("SPLIT_ME_HERE_PLEASE") }
```

Comment:
Yes, that would work.

I think that's even uglier. I tried to be clear about what was going on. My goal was to split the reek output as is. With your change, the REEK_REGEX won't match the entire reek output.

But I can make that change if you want.